src/assets/images/TimeCircleLine.tsx

```diff
@@ -0,0 +1,25 @@
+const TimeCircleLine = () => (
+  <svg
+    width="24"
+    height="24"
+    viewBox="0 0 24 24"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <g clipPath="url(#clip0_1023_8179)">
+      <path d="M13 12V7H11V14H17V12H13Z" fill="currentColor" />
+      <path
+        d="M1 12C1 18.0753 5.92472 23 12 23C18.0753 23 23 18.0753 23 12C23 5.92472 18.0753 1 12 1C5.92472 1 1 5.92472 1 12Z"
+        stroke="currentColor"
+        strokeWidth="2"
+      />
+    </g>
+    <defs>
+      <clipPath id="clip0_1023_8179">
+        <rect width="24" height="24" fill="white" />
+      </clipPath>
+    </defs>
+  </svg>
+);
+
+export default TimeCircleLine;
```

Comment:
@escobarjonatan For sure, we talked about the general direction on one of your recent pull requests, I included some of those figma icons before that convo happened and then started transitioning the best fit MUI icons that I could find without disrupting the flow too much. I will look over once more to remove any icons I happened to add originally but didn't end up using to clean it up a bit.
As @scandycuz mentioned some of the icons just didn't fit in as well with what MUI provided and the icon look wasn't matching to the design, with either color or specific icon images. Hopefully we can find a happy medium in controlling our icons, because I do agree right now in both instances, figma and MUI, they are difficult to customize size/color per current design requirements.

Reply:
Would it also be better to refactor our image assets into an index.ts file? Is there a clean way of identifying which icons are still being referenced?